src/forbids/init.py

```diff
@@ -104,10 +105,13 @@ def generate_series_model(
         grouping_tags.extend(config["instrument"]["version_tags"])
 
     # list all unique instruments and models for this datatype
-    # unique_instruments = bids_layout.__getattr__(f"get_{config['instrument']['uid_tags'][0]}")(**series_entities)
-    instrument_groups = OrderedDict(
-        {tag: getattr(bids_layout, f"get_{tag}")(**series_entities) for tag in grouping_tags}
-    )
+    instrument_groups = OrderedDict()
+
+    for tag in grouping_tags:
+        try:
+            instrument_groups[tag] = getattr(bids_layout, f"get_{tag}")(**series_entities)
+        except Exception as e:
+            lgr.warning(f"Warning the tag {tag} fails to extract from {series_entities} with Exception {e}")
```

Comment:
better not rely on all encompassing Exception with something like .
```suggestion
        getter = f"get_{tag}"
        if hasattr(bids_layout, getter):
            instrument_groups[tag] = getattr(bids_layout, getter)(**series_entities)
```